src/plans/list/components/PlansEmptyState.tsx

```diff
@@ -32,51 +45,58 @@ const PlansEmptyState: FC<{ namespace: string }> = ({ namespace }) => {
   });
 
   return (
-    <ForkliftEmptyState
-      icon={PlusCircleIcon}
-      title={
-        namespace ? (
-          <ForkliftTrans>
-            No plans found in the project <strong>{namespace}</strong>
-          </ForkliftTrans>
-        ) : (
-          t('No plans found')
-        )
-      }
-      textContent={
-        hasSufficientProviders ? (
+    <EmptyState>
+      <EmptyStateHeader
+        titleText={
+          namespace ? (
+            <ForkliftTrans>
+              No Plans found in namespace <strong>{namespace}</strong>.
+            </ForkliftTrans>
+          ) : (
+            t('No Plans found')
```

Comment:
```suggestion
            t('No plans found')
```

Reply:
Again, in all places we use upper case for the entity name since it's a term..
No Providers found
No Mappings found
No Network maps found
No Storage maps found...

I can change it in all places if you think it's incorrect.
CC:// @RichardHoch ?